### ISSUE TITLE
Using next.js SSR and change version from 7.4.23 => 7.4.26

### ISSUE THREAD

**Issue by jackdon** [react-component/menu#275]
```shell
> Ready on http://localhost:3001
(node:24) UnhandledPromiseRejectionWarning: ReferenceError: window is not defined
    at /app/node_modules/ismobilejs/dist/isMobile.min.js:1:1327
    at Object.<anonymous> (/app/node_modules/ismobilejs/dist/isMobile.min.js:1:1349)
    at Module._compile (internal/modules/cjs/loader.js:701:30)
    at Object.Module._extensions..js (internal/modules/cjs/loader.js:712:10)
    at Module.load (internal/modules/cjs/loader.js:600:32)
    at tryModuleLoad (internal/modules/cjs/loader.js:539:12)
    at Function.Module._load (internal/modules/cjs/loader.js:531:3)
    at Module.require (internal/modules/cjs/loader.js:637:17)
    at require (internal/modules/cjs/helpers.js:22:18)
    at Object.<anonymous> (/app/node_modules/rc-menu/lib/util.js:17:16)
```

**Comment by shaoxi2093** [comment]
same +1

**Comment by zhangking** [comment]
@afc163 please

**Comment by afc163** [comment]
Fixed in 7.4.28
